--- ./binutils-2.35.1/bfd/config.bfd.bak	2026-10-01 15:00:50.388511848 +0000
+++ ./binutils-2.35.1/bfd/config.bfd	2026-10-01 15:11:49.776136372 +0000
@@ -223,6 +223,13 @@
 case "${targ}" in
 # START OF targmatch.h
 #ifdef BFD64
+  x86_64-*-trazelos*)
+    targ_defvec=x86_64_elf64_vec
+    targ_selvecs=i386_elf32_vec
+    want64=true
+    ;;
+#endif
+#ifdef BFD64
   aarch64-*-darwin*)
     targ_defvec=aarch64_mach_o_vec
     targ_selvecs="arm_mach_o_vec mach_o_le_vec mach_o_be_vec mach_o_fat_vec"
